experimental/saiexperimentaldashoutboundrouting.h

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright (c) 2014 Microsoft Open Technologies, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *    not use this file except in compliance with the License. You may obtain
+ *    a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    THIS CODE IS PROVIDED ON AN *AS IS* BASIS, WITHOUT WARRANTIES OR
+ *    CONDITIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT
+ *    LIMITATION ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS
+ *    FOR A PARTICULAR PURPOSE, MERCHANTABILITY OR NON-INFRINGEMENT.
+ *
+ *    See the Apache Version 2.0 License for specific language governing
+ *    permissions and limitations under the License.
+ *
+ *    Microsoft would like to thank the following companies for their review and
+ *    assistance with these files: Intel Corporation, Mellanox Technologies Ltd,
+ *    Dell Products, L.P., Facebook, Inc., Marvell International Ltd.
+ *
+ * @file    saiexperimentaldashoutboundrouting.h
+ *
+ * @brief   This module defines SAI P4 extension  interface
+ */
+
+#if !defined (__SAIEXPERIMENTALDASHOUTBOUNDROUTING_H_)
+#define __SAIEXPERIMENTALDASHOUTBOUNDROUTING_H_
+
+#include <saitypes.h>
+
+/**
+ * @defgroup SAIEXPERIMENTALDASH_OUTBOUND_ROUTING SAI - Extension specific API definitions
+ *
+ * @{
+ */
+
+/**
+ * @brief Attribute data for #SAI_OUTBOUND_ROUTING_ENTRY_ATTR_ACTION
+ */
+typedef enum _sai_outbound_routing_entry_action_t
+{
+    SAI_OUTBOUND_ROUTING_ENTRY_ACTION_ROUTE_VNET,
+
+    SAI_OUTBOUND_ROUTING_ENTRY_ACTION_ROUTE_VNET_DIRECT,
+
+    SAI_OUTBOUND_ROUTING_ENTRY_ACTION_ROUTE_DIRECT,
+
+    SAI_OUTBOUND_ROUTING_ENTRY_ACTION_DROP,
```

Comment:
The DROP action is not needed for direction, and should be removed.

Reply:
@marian-pritsak  Hi Marian, Could you please re generate the DASH SAI APIs based on the current P4. Thanks

Reply:
Done